packages/medusa/src/services/order-edit.ts

```diff
@@ -382,4 +386,33 @@ export default class OrderEditService extends TransactionBaseService {
 
     return orderEdit
   }
+
+  async deleteItemChange(itemChangeId: string): Promise<void> {
```

Comment:
thought: we could consider allowing this method to take `orderEditId` as the first param and then doing:
1. Retrieve change
2. Check orderEditId exists
3. Check change.order_edit_id === orderEditId
  a. if orderEditId exists -> throw invalid changeId doesn't belong to orderEditId

Reply:
👍